[2024-05-28 09:34:24.591] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:34:56.653] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:35:28.711] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:36:00.744] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:36:32.787] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:37:04.862] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:37:36.919] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:38:08.940] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:38:40.962] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:39:13.017] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:39:45.056] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:40:17.106] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:40:49.128] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:41:21.168] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:41:53.220] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:42:25.252] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:42:57.284] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:43:29.334] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:44:01.398] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:44:33.452] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:45:05.485] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:45:37.528] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:46:09.548] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:46:41.582] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:47:13.618] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:47:45.681] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:48:17.723] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:48:49.747] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:49:21.806] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:49:53.850] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:52:21.391] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:52:53.437] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:53:25.516] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:53:57.559] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:54:29.591] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:55:01.617] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:55:33.672] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:56:05.713] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:56:37.788] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:57:09.805] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:57:41.834] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:58:13.872] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:58:45.902] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:59:17.940] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 09:59:49.959] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:00:22.026] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:00:54.053] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:01:26.090] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:01:58.125] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:02:30.163] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:03:02.213] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:03:34.233] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:03:51.487] [ERROR] [main] [cn.xuyanwu.spring.file.storage.FileStorageService] [?] [?] - 销毁存储平台 local-plus-1 成功
[2024-05-28 10:03:51.487] [ERROR] [main] [cn.xuyanwu.spring.file.storage.FileStorageService] [?] [?] - 销毁存储平台 minio-1 成功
[2024-05-28 10:03:51.659] [ERROR] [main] [o.s.b.diagnostics.LoggingFailureAnalysisReporter] [?] [?] - 

***************************
APPLICATION FAILED TO START
***************************

Description:

Web server failed to start. Port 33000 was already in use.

Action:

Identify and stop the process that's listening on port 33000 or configure this application to listen on another port.

[2024-05-28 10:04:06.263] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:04:38.296] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:05:10.320] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:05:42.384] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:06:14.421] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:06:46.451] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:07:18.496] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:07:50.535] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:08:22.564] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:08:54.608] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:09:26.653] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:09:58.704] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:10:30.746] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:11:02.823] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:11:34.844] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:12:06.886] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:12:38.931] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:13:10.955] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:13:43.010] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:14:15.055] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:14:47.117] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:15:19.186] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:15:51.231] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:16:23.285] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:16:55.337] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:17:27.371] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:17:59.417] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:18:31.469] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:19:03.535] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:19:35.622] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:20:07.656] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:20:39.694] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:21:11.743] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:21:43.784] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:22:15.828] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:22:47.865] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:23:19.908] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:23:51.937] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:24:23.986] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:24:56.023] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:25:08.484] [ERROR] [main] [cn.xuyanwu.spring.file.storage.FileStorageService] [?] [?] - 销毁存储平台 local-plus-1 成功
[2024-05-28 10:25:08.484] [ERROR] [main] [cn.xuyanwu.spring.file.storage.FileStorageService] [?] [?] - 销毁存储平台 minio-1 成功
[2024-05-28 10:25:08.573] [ERROR] [main] [o.s.b.diagnostics.LoggingFailureAnalysisReporter] [?] [?] - 

***************************
APPLICATION FAILED TO START
***************************

Description:

Web server failed to start. Port 33000 was already in use.

Action:

Identify and stop the process that's listening on port 33000 or configure this application to listen on another port.

[2024-05-28 10:25:11.437] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xede968b6, L:/192.168.1.120:57618 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.453] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x1ed02249, L:/192.168.1.120:57619 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.453] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xe7a86d41, L:/192.168.1.120:57620 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.453] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xd838e1e2, L:/192.168.1.120:57622 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.453] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xf17a3cae, L:/192.168.1.120:57621 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.453] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xffef0ecf, L:/192.168.1.120:57624 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.485] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xdb047a5f, L:/192.168.1.120:57623 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.485] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xdc0c22e6, L:/192.168.1.120:57625 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.485] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xdb11d038, L:/192.168.1.120:57626 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.485] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x95a9b91e, L:/192.168.1.120:57628 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.485] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x63dc12ee, L:/192.168.1.120:57627 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.580] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x32d0acf9, L:/192.168.1.120:57630 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.580] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x59de99dc, L:/192.168.1.120:57629 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.580] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x90250fa7, L:/192.168.1.120:57632 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.678] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x0a46253c, L:/192.168.1.120:57633 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.678] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x10fd876d, L:/192.168.1.120:57634 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.678] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x6c9c73ca, L:/192.168.1.120:57636 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.783] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x8f6f760b, L:/192.168.1.120:57637 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.783] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xe93a2670, L:/192.168.1.120:57635 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.785] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x8a7af69d, L:/192.168.1.120:57639 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.786] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0x50e5d6ce, L:/192.168.1.120:57638 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.786] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xa790b3c7, L:/192.168.1.120:57641 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.786] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xa69d8ea5, L:/192.168.1.120:57640 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:11.878] [ERROR] [redisson-timer-4-1] [org.redisson.client.handler.PingConnectionHandler] [?] [?] - Unable to send PING command over channel: [id: 0xf3998543, L:/192.168.1.120:57642 - R:39.99.169.115/39.99.169.115:6379]
org.redisson.client.RedisTimeoutException: Command execution timeout for command: (PING), params: [], Redis client: [addr=redis://39.99.169.115:6379]
	at org.redisson.client.RedisConnection.lambda$async$0(RedisConnection.java:244)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
[2024-05-28 10:25:28.100] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:26:00.160] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:26:32.199] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:29:13.982] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:29:46.036] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:30:18.097] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:30:50.140] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:31:22.191] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:31:54.227] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:32:26.252] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:32:58.266] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:33:30.310] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:34:02.353] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:34:34.403] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:36:33.759] [ERROR] [main] [cn.xuyanwu.spring.file.storage.FileStorageService] [?] [?] - 销毁存储平台 local-plus-1 成功
[2024-05-28 10:36:33.759] [ERROR] [main] [cn.xuyanwu.spring.file.storage.FileStorageService] [?] [?] - 销毁存储平台 minio-1 成功
[2024-05-28 10:36:35.750] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:36:37.791] [ERROR] [xxl-job, executor ExecutorRegistryThread] [com.xxl.job.core.util.XxlJobRemotingUtil] [?] [?] - 调度服务器连接失败：Connection refused: connect
[2024-05-28 10:36:37.876] [ERROR] [main] [o.s.b.diagnostics.LoggingFailureAnalysisReporter] [?] [?] - 

***************************
APPLICATION FAILED TO START
***************************

Description:

Web server failed to start. Port 3000 was already in use.

Action:

Identify and stop the process that's listening on port 3000 or configure this application to listen on another port.

